specs/agents/README.md

```diff
@@ -44,6 +44,7 @@ You can find details about each of these in the [APM Data Model](https://www.ela
   - [Distributed tracing](tracing-distributed-tracing.md)
   - [Tracer API](tracing-api.md)
   - Instrumentation
+      - [AWS](tracing-instrumentation-aws.md)
```

Comment:
It it unrelated to this change, but I added it intentionally because that is the only "tracing-instrumentation-*.md" file *not* in this table of contents.

Reply:
I added this line separately in https://github.com/elastic/apm/commit/051add6e14fdb86768231d0f6541c2c67758e8b6